tests/test_fuzz.py

```diff
@@ -169,7 +169,7 @@ def test_invalid_Error_WHEN_cli_called_GIVEN_invalid_fuzz_input(self, text):
 		self.assertTrue(theResult, fail_fixture)
 
 	@given(st.text(alphabet=string.ascii_letters + string.digits, min_size=56, max_size=2048))
-	@settings(deadline=None)
+	@settings(deadline=2222)
```

Comment:
🤓 After gathering metrics from local testing, it seems under test conditions `2200 ms` is enough time for this fuzz test, so with a nice overhead of 22 this should strike a nice balance between allowing slower tests like this, but still limiting the total run time to fail hung tests.

Similarly above around line 129